# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: js/src/wasm/WasmInstance.h
# Commit: ce799cb566ac
# Full Hash: ce799cb566ac0fb95468118608132be11db6a359
# Author: Ryan Hunt <rhunt@eqrion.net>
# Date: 2022-03-24 21:54:20
# Regressor Bug: 1642412
# File Overlap Count: 19
# Description:
#   Bug 1642412 - wasm: Rollup of inline table.get/set for anyref. r=lth
#   
#   Rollup of the previous inline table.get/set patches.
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D141024
# ==============================================================================

diff -r c10609d434ae -r ce799cb566ac js/src/wasm/WasmInstance.h
--- a/js/src/wasm/WasmInstance.h	Thu Mar 24 17:40:32 2022 +0000
+++ b/js/src/wasm/WasmInstance.h	Thu Mar 24 17:56:50 2022 +0000
@@ -431,13 +431,12 @@
                            uint32_t dstTableIndex, uint32_t srcTableIndex);
   static int32_t tableFill(Instance* instance, uint32_t start, void* value,
                            uint32_t len, uint32_t tableIndex);
-  static void* tableGet(Instance* instance, uint32_t index,
-                        uint32_t tableIndex);
+  static void* tableGetFunc(Instance* instance, uint32_t index,
+                            uint32_t tableIndex);
   static uint32_t tableGrow(Instance* instance, void* initValue, uint32_t delta,
                             uint32_t tableIndex);
-  static int32_t tableSet(Instance* instance, uint32_t index, void* value,
-                          uint32_t tableIndex);
-  static uint32_t tableSize(Instance* instance, uint32_t tableIndex);
+  static int32_t tableSetFunc(Instance* instance, uint32_t index, void* value,
+                              uint32_t tableIndex);
   static int32_t tableInit(Instance* instance, uint32_t dstOffset,
                            uint32_t srcOffset, uint32_t len, uint32_t segIndex,
                            uint32_t tableIndex);